src/utils/daos_hdlr.h

```diff
@@ -190,6 +191,14 @@ int pool_autotest_hdlr(struct cmd_args_s *ap);
  * int pool_stat_hdlr(struct cmd_args_s *ap);
  */
 
+/* general datamover operations */
+void dm_cont_free_usr_attrs(int n, char ***_names, void ***_buffers, size_t **_sizes);
+int dm_cont_get_usr_attrs(struct cmd_args_s *ap, daos_handle_t coh, int *_n, char ***_names,
+			  void ***_buffers, size_t **_sizes);
+int dm_cont_get_all_props(struct cmd_args_s *ap, daos_handle_t coh, daos_prop_t **_props,
+  			  bool get_oid, bool get_label, bool get_roots);
```

Comment:
(style)  code indent should use tabs where possible
(style)  please, no space before tabs